Shared/Models/WriteFreelyModel.swift

```diff
@@ -58,6 +57,42 @@ extension WriteFreelyModel {
         guard let loggedInClient = client else { return }
         loggedInClient.getUserCollections(completion: fetchUserCollectionsHandler)
     }
+
+    func fetchUserPosts() {
+        guard let loggedInClient = client else { return }
+        loggedInClient.getPosts(completion: fetchUserPostsHandler)
+    }
+
+    func publish(post: Post) {
+        guard let loggedInClient = client else { return }
+
+        if post.wfPost == nil {
+            // This is a new local draft.
+            post.wfPost = WFPost(
+                body: post.body,
+                title: post.title,
+                createdDate: post.createdDate
```

Comment:
Can we also include the `lang` and `rtl` values here (only upon publishing a new post)?

The former could ideally be set by the system language for now (user-configurable in the future), and the latter set based on whether or not the language is written right-to-left ([example helper code here](https://github.com/writeas/web-core/blob/master/i18n/rtl.go)).

Reply:
Good call! We can use built-in Swift methods for this — no need for helper code here.

- `Locale.current.languageCode` is an Optional String for setting `lang`
- `Locale.characterDirection(forLanguage:)` returns a `Locale.LanguageDirection` case for setting `rtl`

(There's also `Locale.lineDirection` for top-to-bottom vs. bottom-to-top)

@thebaer How about we go ahead with this PR as is, since other PRs branch off of this work, and I open an issue to make this quick fix in a subsequent PR?

Reply:
Perfect, sounds good :+1:

Reply:
Opened #35 to track this.